src/main/resources/templates/index.html

```diff
@@ -1,13 +1,14 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <link rel="stylesheet" href="../static/index.css">
-</head>
-<body>
+    <link rel="stylesheet" href="index.css">
     <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDoEEepG0o3JkfmejyT-bn3S9EPLJHCLAI&callback=initMap"
-type="text/javascript"></script>
+	type="text/javascript"></script>
 
-    <script src="../static/index.js"></script>
+    <script src="index.js"></script>
+</head>
+<body>
+    

```

Comment:
moved the script to the head, changed the path of the css and js files so that they are correct and get accessed